content/en/index.md

```diff
@@ -14,9 +14,9 @@ It’s free to use for all developers and software providers, with Sigstore’s
 
 ## How to use Sigstore
 
-* Sign software artifacts with [Cosign](/cosign/overview/) — [Quick start](/cosign/overview/#quick-start)
+* Install Cosign - [Cosign Installation page](/cosign/installation/)
+* Sign and verify software artifacts with [Cosign](/cosign/overview/) — [Quick start](/cosign/overview/#quick-start)
 * Sign Git commits with [Gitsign](/gitsign/overview/) - [Quick start](/gitsign/overview/#quick-start)
-* Verify entries with [Cosign](/cosign/verify/)
 * Use the [policy controller](/policy-controller/overview/) to enforce Kubernetes policies
 * Use the [Fulcio Certificate Authority](/fulcio/overview/)
```

Comment:
Can we remove the Fulcio bullet point? I think this section works better if it's focused only on clients.

Reply:
OK...."and verify" part of the sentence is removed.  @olivekl